src/index.css

```
@@ -100,15 +116,15 @@
 
   body {
     @apply min-h-screen m-0;
-    background-color: rgb(var(--surface-base));
+    background-color: rgb(var(--surface-app-frame));
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)